app/dashboard/templatetags/bundle.py

```diff
@@ -38,8 +38,8 @@
 """
 
 
-# check for production env
-isProduction = settings.ENV in ['prod']
+# check if assets are hosted in the env
+assetsHosted = settings.ENV in ['prod', 'test']
```

Comment:
Can we alter this to `isDevelopment` and negate the `in` check - I feel like `assetsHosted` will have the reader asking "where are the assets hosted?"